package.json

```diff
@@ -14,8 +14,8 @@
   "version": "3.18.0",
   "scripts": {
     "dtslint": "dtslint types/getstream",
-    "lint": "./node_modules/.bin/prettier --no-config --list-different './{src,test}/**' && ./node_modules/.bin/eslint --max-warnings 0 './{src,test}/**'",
-    "lint-fix": "./node_modules/.bin/prettier --no-config --write './{src,test}/**' && ./node_modules/.bin/eslint './{src,test}/**' --fix",
+    "lint": "./node_modules/.bin/prettier --list-different '**/*.{js,ts}' && ./node_modules/.bin/eslint '**/*.js' --max-warnings 0",
+    "lint-fix": "./node_modules/.bin/prettier --write '**/*.{js,ts}' && ./node_modules/.bin/eslint --fix '**/*.js'",
```

Comment:
I keep going back and forth on whether I rather be explicit on the command line or leave this in the .(eslint|prettier)ignore files. I'm currently thinking this is a good change as it'll make it less likely that the linter will miss any new sibling folders should they be added alongside `src` and `tst`. If you have other good pros/cons I'd be interested to discuss/learn.